src/wizard/wizard.scss

```diff
@@ -2,168 +2,147 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards {
+@mixin iui-wizard {
   @include iui-reset;
   @include iui-font-family;
 
-  > .iui-wizards-wrapper {
+  > .iui-wizard-container {
```

Comment:
Hm, is this class needed? Can't it be merget with the root one?

Reply:
I don't believe the container can be merged with the parent.  The parent contains an `<ol class="iui-wizard-container">` (in red) as well as `<p class="iui-wizard-steps">` (in pink).
![Screen Shot 2021-08-04 at 2 02 56 PM](https://user-images.githubusercontent.com/849817/128231580-6b9f30b6-4641-4d43-8643-e6b1286efbd5.png)

Reply:
Missed that part.